src/main/java/com/mateacademy/springmvc/config/ApplicationConfig.java

```diff
@@ -0,0 +1,72 @@
+package com.mateacademy.springmvc.config;
+
+import org.hibernate.jpa.HibernatePersistenceProvider;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+
+import java.util.Properties;
+
+@Configuration
+@EnableJpaRepositories("com.mateacademy.springmvc.repository")
+@ComponentScan("com.mateacademy.springmvc")
+@PropertySource("application.properties")
+public class ApplicationConfig {
+    @Value("${url}")
+    private String url;
+
+    @Value("${user}")
+    private String user;
+
+    @Value("${driver}")
+    private String driver;
+
+    @Value("${password}")
+    private String password;
+
+    @Value("${hibernate.dialect}")
+    private String dialect;
+
+    @Value("${ddl-auto}")
+    private String ddlAuto;
+
+    @Bean
+    public DriverManagerDataSource dataSource() {
+        DriverManagerDataSource dataSource = new DriverManagerDataSource(url, user, password);
+        dataSource.setDriverClassName(driver);
+        return dataSource;
+    }
+
+    @Bean
+    public LocalContainerEntityManagerFactoryBean entityManagerFactory() {
```

Comment:
Minor: Could use builder here?
https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/orm/jpa/EntityManagerFactoryBuilder.html

Reply:
for what? with this builder i cant realize all methods which i need) for example "setPackagesToScan"